src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -259,7 +269,11 @@ protected override void Dispose(bool disposing)
         /// transaction.
         /// </remarks>
         public new virtual SqliteCommand CreateCommand()
-            => new SqliteCommand { Connection = this, Transaction = Transaction };
+        {
+            var command = new SqliteCommand { Connection = this, Transaction = Transaction };
+            _commands.Add(new WeakReference<SqliteCommand>(command));
```

Comment:
This list needs to be maintained by the `SqliteCommand.Connection` property. Imagine the following.

```c#
command.Connection = new SqliteConnection("...");
```